Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -290,17 +326,17 @@ open class ChatChannelViewModel: ObservableObject, MessagesDataSource {
     }
     
     public func onViewAppear() {
-        setActive()
-        messages = channelDataSource.messages
-        checkNameChange()
```

Comment:
The above two lines are the things we need to perform when this model becomes active (`isActive == true`).